### PR TITLE
Correct table formatting and article formatting & organization

### PR DESCRIPTION
**Pull request by bpaduch**
Improve document's:
- table formatting
- formatting and callouts
- remove manual TOC and use auto TOC that appears in ampproject.org via `[TOC]`
- minor wording fixes

Fixes https://github.com/ampproject/docs/issues/435

### CODE REVIEW

File: extensions/amp-bind/amp-bind.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # <a name="amp-bind"></a> `amp-bind`
 
-**This extension is under active development, and the version number of the specification section should provide guidance to its evolution.**
+{% call callout('Important', type='caution') %}
```
Comment (choumx) [inline extensions/amp-bind/amp-bind.md]: Nice! Will these only work on ampproject.org or also on GitHub too?

File: extensions/amp-bind/amp-bind.md [hunk 2]
```diff
@@ -218,7 +195,7 @@ Expressions are similar to JavaScript with some important differences.
 
 - Expressions may only access the containing document's [state](#state).
 - Expressions **do not** have access to globals like `window` or `document`.
-- Only [whitelisted functions](#whitelisted-functions) are allowed.
+- Only [whitelisted functions](#white-listed-functions) are allowed.
```
Comment (choumx) [inline extensions/amp-bind/amp-bind.md]: Also change the link text?

File: extensions/amp-bind/amp-bind.md [hunk 3]
```diff
@@ -51,70 +53,40 @@ limitations under the License.
   </tr>
 </table>
 
+[TOC]
```
Comment (choumx) [inline extensions/amp-bind/amp-bind.md]: Cool!

File: extensions/amp-bind/amp-bind.md (hunk 1 above)
Comment (bpaduch) [inline extensions/amp-bind/amp-bind.md]: Only on ampproject.org.

**Pull request merged**